apps/frontend/src/app/poll/choose-events/choose-events.component.html

```diff
@@ -35,11 +35,16 @@ <h1 class="text-center">
       {{ closedReason }}
     </div>
   }
-  @if (poll?.settings?.blindParticipation && !showResults) {
+  @if (!this.userVoted() && !this.isAdmin && this.poll?.settings?.showResult === ShowResultOptions.AFTER_PARTICIPATING) {
     <div class="alert alert-info">
       This is a blind poll. You can't see results or other user's votes until you participate yourself.
     </div>
   }
+  @if (showNever) {
```

Comment:
I'm still not happy with this. Maybe this can be done with a @switch on showResult and extra @if for the AFTER_PARTICIPATING case. After all these two messages should never appear at the same time.

Or, alternatively, add a variable `hiddenReason: string` like `closedReason` and set it in the updateHelpers method.